packages/cli/src/workflows/workflows.controller.ee.ts

```diff
@@ -213,7 +216,7 @@ EEWorkflowController.patch(
 	'/:id(\\d+)',
 	ResponseHelper.send(async (req: WorkflowRequest.Update) => {
 		const { id: workflowId } = req.params;
-		// const forceSave = req.query.forceSave === 'true'; // disabled temporarily - tests were also disabled
+		const forceSave = req.query.forceSave === 'true';
```

Comment:
If you can please unskip and update the relevant tests, and ensure they pass on all three DBs.

Reply:
Thanks for reminding me this, tests uncommented and fixed.